cmd/registrytoken/registrytoken.go

```diff
@@ -54,19 +55,41 @@ More info about docker credentials helpers here: https://github.com/docker/docke
 		Args:      cobra.ExactValidArgs(1),
 	}
 
-	cmd.RunE = func(_ *cobra.Command, args []string) error {
-		action := args[0]
+	cmd.Run = func(_ *cobra.Command, args []string) {
 		ctx := context.Background()
 		if err := contextCMD.NewContextCommand().Run(ctx, &contextCMD.ContextOptions{}); err != nil {
-			return err
+			_, _ = fmt.Fprintln(os.Stdout, err)
+			os.Exit(1) // skipcq: RVV-A0003
 		}
 		conf := okteto.Config{}
 		if !conf.IsOktetoCluster() {
-			return errors.ErrContextIsNotOktetoCluster
+			_, _ = fmt.Fprintln(os.Stdout, errors.ErrContextIsNotOktetoCluster)
+			os.Exit(1) // skipcq: RVV-A0003
 		}
 		h := dockercredentials.NewOktetoClusterHelper(regCreds{conf})
-		return credentials.HandleCommand(h, action, os.Stdin, os.Stdout)
+		action := args[0]
+		if err := credentials.HandleCommand(h, action, os.Stdin, os.Stdout); err != nil {
+			_, _ = fmt.Fprintln(os.Stdout, err)
+			os.Exit(1) // skipcq: RVV-A0003
+		}
 	}
 
 	return cmd
 }
+
+func IsRegistryCredentialHelperCommand(args []string) bool {
+	if len(args) != 3 {
+		return false
+	}
+
+	if args[1] != "registrytoken" {
+		return false
+	}
+
+	switch args[2] {
+	case "store", "get", "erase", "list", "version":
```

Comment:
how about storing the valid args in a const and using that const here and in `cobra.Command.ValidArgs`, in command declaration? less error prone I think